src/components/NavbarStudyStudioSVGSelector.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {ReactComponent as Menu} from './icons/menu.svg';
+import {ReactComponent as Close} from './icons/close.svg';
+import {ReactComponent as Home} from './icons/home.svg';
+import {ReactComponent as Categories} from './icons/categories.svg';
+import {ReactComponent as Subscription} from './icons/subscription.svg';
+import {ReactComponent as HatMylesson} from './icons/hat_mylesson.svg';
+import {ReactComponent as WatchLater} from './icons/watch_later.svg';
+import {ReactComponent as MyLesson} from './icons/my_lesson.svg';
+import {ReactComponent as Management} from './icons/management.svg';
+
+type Props = {
+    icon: string;
+}
+
+const NavbarStudyStudioSVGSelector = ({ icon }: Props) => {
+    switch (icon) {
```

Comment:
It is better to create here object lookup for prettier form (instead of 'switch' / 'case'):

const menuItems: {
menu: Menu,
close: Close,
home: Home,
categories: Categories,
...
}

return React.createElement(menuItems[icon])

See example: https://medium.com/@adrian_98730/dynamically-rendering-react-components-376fb3ef7a2a

if you need deafult case use something like this:

if (!menuItems[icon]) {
return <svg />
}

Reply:
But in my case the code gets bigger, maybe I can use it as optional?